nodezator/winman/states/loadedfile.py

```diff
@@ -73,6 +74,14 @@ def loaded_file_event_handling(self):
             if event.type == QUIT:
                 raise QuitAppException
 
+            ### MOUSEWHEEL
+            elif event.type == MOUSEWHEEL:
+                mod = SERVICES_NS.get_pressed_mod_keys()
+                if  mod & KMOD_SHIFT: #KMOD_CTRL:
+                    APP_REFS.ea.scroll(event.y*50, event.x*50)
+                else:
+                    APP_REFS.ea.scroll(event.x*50, event.y*50)
```

Comment:
This solution is mostly fine and even very similar to how I'd do it myself. My only complain is that although clever, swapping the values of event.x and event.y here is not immediately readable (requires a bit of extra thinking to understand, however brief).

IMHO, it is better to be more explicit about it, like this:

```
            elif event.type == MOUSEWHEEL:

                mod = SERVICES_NS.get_pressed_mod_keys()
                x, y = event.x*50, event.y*50

                if  mod & KMOD_SHIFT:
                    x, y = y, x

                APP_REFS.ea.scroll(x, y)
```

This way it is clear that the call to APP_REFS.ea.scroll() is always suitably made with x and y in the position we expect them to be (that is, x, then y). The only difference is that we swap their values beforehand if the "shift" key is pressed.